### DISCUSSION TITLE
What are the benefits and significance of using Preferences()

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#3585] [Q&A] [answered]
## ②What's the advantage of using Preferences()？
```golang
fyne.CurrentApp().Preferences()
fyne.CurrentApp().Preferences().SetString(num, "1")
```

**Comment by andydotxyz** [comment, marked as answer]
The advantage is that you don't have to write any persistence code because it is all provided.
It also supports data binding out of the box.
The data is not limited in any specific way, though it is all stored in a single file if that matters to your usage?

**Reply by junmaqiang** [reply]
thank you it's important
